conf/routes

```diff
@@ -19,7 +19,7 @@ GET    /v3/subpopulations        @org.sagebionetworks.bridge.play.controllers.Su
 POST   /v3/subpopulations        @org.sagebionetworks.bridge.play.controllers.SubpopulationController.createSubpopulation
 GET    /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.getSubpopulation(guid: String)
 POST   /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.updateSubpopulation(guid: String)
-DELETE /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.deleteSubpopulation(guid: String)
+DELETE /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.deleteSubpopulation(guid: String, physical: String ?= "false")
```

Comment:
Would it be cleaner to have logical and physical delete be separate URLs? This would make the controller code much simpler.